_posts/2017-09-05-planning-for-7.0.md

```diff
@@ -0,0 +1,633 @@
+---
+layout: post
+title:  "Planning for 7.0"
+author: Henry Zhu
+date:   2017-09-05 15:00:00
+categories: announcements
+share_text: "Planning for 7.0"
+---
+
+If you didn't know already, we're planning on releasing a 7.0 version soon 🙌 ! Work on it actually started back in February, when I just wanted to make a release to drop Node 0.10/0.12 support and remove babel-runtime and various other code. And since then, we've done releases up to `alpha.20`.
+
+> We're going to update this post throughout the beta releases
+
+Since we're still just a volunteer project, it's been difficult for most of the team to stay focused and motivated to make all these changes and continue to maintain a project that so many companies, bootcamps, and tools rely on so much. But in the meantime we've made a lot of progress: [weekly meetings/notes](https://github.com/babel/notes), participating as invited guests at TC39 for the last few meetings, facilitating in both [RGSoC](https://railsgirlssummerofcode.org/) and [GSoC](https://summerofcode.withgoogle.com), and creating an [Open Collective](https://opencollective.com/babel).
+
+> Upgrading for most projects should be as simple as updating your `package.json` deps to `7.0.0-beta.1`. (we will be pinning to exact versions until the official release). For the whole time we've been working on 7.0, we've been using it in Babel itself (so meta) and at my workplace at Behance.
+
+Example:
+
+```
+{
+  "devDependencies": {
+    "babel-cli": "7.0.0-beta.1"
+  }
+}
+```
+
+I'd like to go over some notable changes for our first beta release (it's still a lot *smaller* in scope in terms of breaking changes than the previous 6.0 release).
+
+## Re-iterating Project Goals
+
+Before we go into that, I just want to repeat again what the purpose of Babel is.
+
+As Babel has been transformed from 6to5, browsers have been implementing more of the spec, and users are more comfortable with using the latest syntax/build tooling, Babel's goals haven't changed too much.
+
+Our two goals are hand in hand: help developers transform new syntax into backwards compatible code (abstract browser support away), and be a bridge to help TC39 get feedback on new ECMAScript proposals and for the community to have a say in the future of the language.
+
+Thus, I think it's an understatement to say that Babel is a vital part of the JavaScript community (almost 10 million downloads a month of `babel-core`) and needs its support. (The only talks I've tried to give are about this point: [JSConf EU](https://github.com/hzoo/maintaining-an-oss-project), [React Rally](https://github.com/hzoo/so-how-does-babel-even-work), [TC39](https://github.com/hzoo/role-of-babel-in-js)). I'll re-iterate a point I made recently: "What happens when Babel dies"? What happens when the current group of people interested in this project get bored/burned out/move on to other things? (What if it's already happened a lot). What are we going to do about it?
```

Comment:
> What happens when Babel dies?

Sounds like it's a question of time (maybe it is), I would rather say: if Babel dies.